packages/application/src/test/application.spec.ts

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as nbformat from '@jupyterlab/nbformat';
+import { NotebookModel } from '@jupyterlab/notebook';
+import { JupyterServer, NBTestUtils } from '@jupyterlab/testutils';
+
+import { NotebookParser } from '../parsing';
+import { RequestHandler } from '../requests';
+import { FrontendServices } from '../services';
+
+const server = new JupyterServer();
+const notebookWithEnvVars: any = {
+  cells: [
+    {
+      cell_type: 'code',
+      execution_count: null,
+      metadata: {},
+      outputs: [],
+      source: [
+        'import os\n',
+        "print(os.environ['HOME'])\n",
+        'print(os.environ["HOME2"])'
```

Comment:
Tested locally and `getenv()` works but `environb` and `getenvb` don't work - I can change the envVars code to include those as well.

Reply:
yeah the util function doesn't support environb and getenvb because I've never heard of those and they never came up on google when I was writing it. It would be simple to add support for them though